core/ingest/src/main/java/org/locationtech/geowave/core/ingest/spi/IngestFormatPluginRegistry.java

```diff
@@ -40,4 +43,16 @@ private static void initPluginProviderRegistry() {
     }
     return pluginProviderRegistry;
   }
+
+  @Override
+  public Map<String, LocalFileIngestPlugin<?>> getDefaultLocalIngestPlugins() {
+    final Map<String, LocalFileIngestPlugin<?>> retVal =
+        new HashMap<>(getPluginProviderRegistry().size());
+    for (final Entry<String, IngestFormatPluginProviderSpi<?, ?>> e : getPluginProviderRegistry().entrySet()) {
+      retVal.put(
+          e.getKey(),
+          e.getValue().createLocalFileIngestPlugin(e.getValue().createOptionsInstances()));
+    }
+    return retVal;
+  }
```

Comment:
This can be a bit cleaner with Java8 streams: `return getPluginProviderRegistry().entrySet().stream().collect(
        Collectors.toMap(
            Entry::getKey,
            e -> e.getValue().createLocalFileIngestPlugin(e.getValue().createOptionsInstances())));`